doc/api/fs.md

```diff
@@ -956,7 +956,7 @@ const defaults = {
 
 `options` may also include a `start` option to allow writing data at
 some position past the beginning of the file.  Modifying a file rather
-than replacing it may require a `flags` mode of `r+` rather than the
+than replacing it may require a `flags` mode of `a` rather than the
```

Comment:
I don't think the current documentation is wrong, if you specify the right `start` option (say, the length of the file) with `flag` set to `r+`, it won't overwrite existing content, which is mentioned above.